Andrei_Bakaushyn/06. Ansible_Workshop/index1.html.j2

```diff
@@ -0,0 +1,5 @@
+<html>
```

Comment:
Please use one template file, 1 and 2 numbers can be fetched from variables, or more simple: you can keep {{ ansible_hostname }} - it will be unique content for your tests